circus-train-integration-tests/src/test/java/com/hotels/bdp/circustrain/integration/CircusTrainParquetSchemaEvolutionIntegrationTest.java

```diff
@@ -0,0 +1,682 @@
+/**
```

Comment:
So is this the class you're not so sure about adding as it increases the test running time so much? One option could be to use a Junit 4 Category (`SchemaTests`) and then turn off this category by default when Maven runs the test. Then in the SCHEMA_EVOLUTION.md you could add a section about the tests and show how one can run them on the CLI by telling Maven to specifically run tests with that category. I assume that's the idea here - they're a safety net that we should never need but they could be useful every now and then just to make sure we haven't broken any of the schema evolution rules as we understand them today?

Reply:
Yep this is the one. In total these tests take ~10 mins on my machine. Plus they all pass (when in reality the non supported schema evolutions shouldn't - for those cases I've made the tests expect an exception to be thrown).

They're a safety net, and also useful if a feature is ever added to CT to support some of the currently failing evolutions. They're essentially supporting tests for the added readme as "proof".

Reply:
I've added junit category as suggested, added to the doc too. Let me know what you think, thanks!